### ISSUE TITLE
New crate versions?

### ISSUE THREAD

**Issue by randombit** [RustCrypto/elliptic-curves#1068]
There have been a couple of pre-releases now, I was wondering what the plan is for a "final" 0.14 of the various crates (mostly concerned with `k256` and `p256` personally). Is there anything I can do to help move things towards a new version? Testing/etc?

**Comment by tarcieri** [comment]
Also if there are specific non-breaking changes you want, we can potentially backport them to the stable release series

**Comment by randombit** [comment]
Main thing that's pressing for us is #1041, which should be pretty easy to backport. I wasn't aware this was an option. Would the idea be to create a new branch off of e158ce5cf0e9acee2fd76aff2a628334f5c771e5, make the change on top of that and publish 0.13.4 from there?

**Comment by tarcieri** [comment]
I think I'd like to make an overall `stable` branch from where we initially switched to prerelease versions and keep changes there (also incorporating previous backports).

It's something we probably should've been doing every time we switch `master` to development.

**Comment by randombit** [comment]
#1072 has a backport of #1041 onto the `k256 0.13.3` commit

**Comment by ycscaly** [comment]
Is `0.14`  planned soon?

**Comment by tarcieri** [comment]
Yes, though as noted earlier there are several prerequisites to release first: https://github.com/RustCrypto/traits/issues/1571

**Comment by tarcieri** [comment]
Coming from https://github.com/RustCrypto/traits/issues/1571 I will try to cut the remaining crates today or tomorrow except for `ed448-goldilocks`.

Regarding `ed448-goldilocks`, it still needs a lot of work. Unlike other crates where we decided not to let perfect be the enemy of done, it really does need a lot of work before I would consider it in a releasable state.

There are already dozens of PRs to do much of the work already open (please do not try to help by vibe coding), I need to go through them though and figure out which ones are showstoppers for the next release, and then do the work to get them merged.

I can make a separate tracking ticket for its release.

**Comment by tarcieri** [comment]
Here is a specific tracking issue for `ed448-goldilocks` v0.14.0: #1907
